qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -958,7 +958,7 @@ def _get_acquire_instruction_timings(
 
 
 def _to_schedule_list(
-    run_input: List[Union[QuantumCircuit, Schedule, ScheduleBlock]], backend: BackendV2
+    run_input: List[Union[QuantumCircuit, Schedule, ScheduleBlock]], backend: BackendV1
```

Comment:
This is actually called only on `DynamicsBackend`, which is `BackendV2`. I'm assuming it works on `DynamicsBackend` otherwise `DynamicsBackend` would never work. Is this issue in Terra in the `main` branch or on a current release?

Reply:
Oh, sorry I misunderstood it. We cannot change this type hint.
Anyway, `DynamicsBackend` would never work if it has the same architecture as backendV2. I haven't solved this issue in the current release 0.25 because I found other bugs in the current code in terra.  However, I am not sure if `DynamicsBackend` would not work in fact because for example, `DynamicsBackend`  enable to measure all qubits independently which is different from the real ibm backend.

p.s.
Scheduling with backendV2 needs `meas_map` and `DynamicsBackend`  has to have `meas_map`.